ui/src/app/edge/live/flat/abstract-flat-widget.ts

```diff
@@ -28,6 +28,7 @@ export abstract class AbstractFlatWidget implements OnInit, OnDestroy {
         @Inject(Websocket) protected websocket: Websocket,
         @Inject(ActivatedRoute) protected route: ActivatedRoute,
         @Inject(Service) protected service: Service,
+        @Inject(Utils) public utils: Utils,
```

Comment:
`Utils` are only static methods, so it should not be required to `Inject` it. Injection is only required for services that keep a context/state.

Reply:
Static methods should be used in a static way and not with an instance + inject. There should actually never exist an instance of the `Utils` class.